backstop/tests/text-field.html

```diff
@@ -29,6 +29,16 @@
         flex-direction: column;
         gap: 11px;
       }
+
+      #block-labels > .iui-input-container { margin-bottom: 11px; }
+
+      #inline-labels {
+        display: grid;
+        row-gap: 11px;
```

Comment:
why not use iui-baseline?

Reply:
it's in the `head` of the html file, you can't use Sass variables here and it also doesn't matter because it's not part of the component